src/views/Home2/components/StatsSection/helpers.tsx

```diff
@@ -0,0 +1,28 @@
+import { request, gql } from 'graphql-request'
+import { BITQUERY_API } from 'config/constants/endpoints'
+
+export const getUsersAndTrades = async (): Promise<{ addressCount: number; txCount: number }> => {
+  try {
+    const response = await request(
+      BITQUERY_API,
+      gql`
+        query getLotteries {
+          ethereum(network: bsc) {
+            dexTrades(
+              exchangeAddress: { is: "0xca143ce32fe78f1f7019d7d551a6402fc5350c73" }
+              date: { since: "2021-06-26" }
```

Comment:
Temporary solution until we figure out the best production-ready way to get this data (used for these two):

![Screenshot 2021-07-27 at 22 45 04](https://user-images.githubusercontent.com/79279477/127231696-4b792e46-5bcb-437a-a793-c53357a24c82.png)

I'll make this value dynamic if we do end up using BITQUERY...